runtime/bcutil/cfreader.c

```diff
@@ -1514,6 +1514,8 @@ checkMethods(J9CfrClassFile* classfile, U_8* segment, U_32 flags)
 	BOOLEAN nameIndexOK;
 	U_32 classfileVersion = flags & BCT_MajorClassFileVersionMask;
 
+	PORT_ACCESS_FROM_PORT(portLib);
```

Comment:
Why does this take a portlb and set up for using it?  I don't see a use in the function.

Reply:
That was just for debugging.